src/DoubleTheDonation/FormExtension/Actions/LoadAssets.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace GiveDoubleTheDonation\DoubleTheDonation\FormExtension\Actions;
+
+use GiveDoubleTheDonation\DoubleTheDonation\Helpers\DoubleTheDonationApi;
+
+/**
+ * @unreleased
+ */
+class LoadAssets
+{
+    /**
+     * @var DoubleTheDonationApi
+     */
+    private $api;
+
+    /**
+     * @param DoubleTheDonationApi $api
+     */
+    public function __construct(DoubleTheDonationApi $api)
+    {
+        $this->api = $api;
+    }
+
+    /**
+     * Load Form builder block
+     *
+     * @unreleased
+     */
+    public function formBuilder(): void
+    {
+        wp_enqueue_script(
+            'givewp-form-extension-dtd-block',
+            GIVE_DTD_URL . 'build/block.js',
```

Comment:
whenever using wp scripts we'll need to add the appropriate dependencies so that `@wordpress` packages are being loaded from WordPress. you can use the `ScriptAsset::get($path)` helper we have.